include/z64save.h

```diff
@@ -1108,10 +1108,10 @@ typedef enum {
 #define WEEKEVENTREG_60_01 PACK_WEEKEVENTREG_FLAG(60, 0x01)
 #define WEEKEVENTREG_60_02 PACK_WEEKEVENTREG_FLAG(60, 0x02)
 #define WEEKEVENTREG_60_04 PACK_WEEKEVENTREG_FLAG(60, 0x04)
-#define WEEKEVENTREG_60_08 PACK_WEEKEVENTREG_FLAG(60, 0x08)
-#define WEEKEVENTREG_60_10 PACK_WEEKEVENTREG_FLAG(60, 0x10)
+#define WEEKEVENTREG_60_08 PACK_WEEKEVENTREG_FLAG(60, 0x08) // Seen the mayor's meeting
+#define WEEKEVENTREG_60_10 PACK_WEEKEVENTREG_FLAG(60, 0x10) // Completed mayor meeting event
 #define WEEKEVENTREG_60_20 PACK_WEEKEVENTREG_FLAG(60, 0x20)
-#define WEEKEVENTREG_60_40 PACK_WEEKEVENTREG_FLAG(60, 0x40)
+#define WEEKEVENTREG_60_40 PACK_WEEKEVENTREG_FLAG(60, 0x40) // Talked to mayor on final night
```

Comment:
I think we could try naming those flags.
What do you think about this:
- `WEEKEVENTREG_60_08`: `WEEKEVENTREG_ATTENDED_MAYOR_DOTOUR_MEETING`
- `WEEKEVENTREG_60_10`: A bit tricky, maybe `WEEKEVENTREG_ENDED_MAYOR_DOTOUR_MEETING`?
- `WEEKEVENTREG_60_40`: `WEEKEVENTREG_TALKED_MAYOR_DOTOUR_NIGHT_3`

We could possibly drop either the `MAYOR` or `DOTOUR` part, since it is a bit redundant

Reply:
Yep, I can definitely rename these. Originally I didn't because it looked like most were the default naming pattern, and I wasn't sure if that was preferred. I suppose that could just be because a lot of the code is still undocumented?

Reply:
How about this? I analyzed their usage and realized that  `WEEKEVENTREG_60_10` is actually only used in one place internally in the actor to handle what I can only imagine is an edge case, so I ended up renaming that to `WEEKEVENTREG_RECEIVED_MAYOR_REWARD`. 

However, `WEEKEVENTREG_63_80` is more widely referenced by several actors to check whether the event has been completed, so I changed that to `WEEKEVENTREG_RESOLVED_MAYOR_MEETING` (and left the comment about the Couple's Mask trigger for clarity). Thoughts?